api/src/main/resources/hibernate.cfg.xml

```diff
@@ -23,7 +23,7 @@
 		<!-- API -->
 
 		<mapping resource="org/openmrs/api/db/hibernate/AllergyReaction.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Concept.hbm.xml" />
+		<!-- <mapping resource="org/openmrs/api/db/hibernate/Concept.hbm.xml" /> -->
```

Comment:
do you still need Concept.hbm.xml in any place,we shouldnt need this comment line

Reply:
No, but I left it commented because I already have some errors to resolve and I will delete it after passing the tests.